### DISCUSSION TITLE
Regarding latest attack on CRYSTALS-Kyber

### DISCUSSION THREAD

**Discussion by Solymanx7** [pkic/community#27] [Post-Quantum Cryptography] [answered]
Regarding the side-channel attack using recursive deep learning on CRYSTALS-Kyber that was able to expose both session and long term keys, Is there any plan to discuss the modifications required to avoid such attacks or any suggestive actions taken into standardizing new applied ones in NIST ?

Thanks a lot

**Comment by primetomas** [comment, marked as answer]
I'd say countermeasure against side channel attacks is something implementations work with continuously. I don't consider it specific for PQC, and it's unlikely to be a modification to the algorithm itself but an implementation detail. It's expected that early implementations are not fully hardened yet, and I know that work is going on in libraries implementing the new algorithms to harden against side channel attacks, using deep-learning and classic side channel attacks. 
I don't think it is a topic for the PKI Consortium imho. NIST forums are probably a better place for such discussions.

Cheers,
Tomas
